sdk/batch/Azure.Compute.Batch/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Security.Cryptography;
+
+[assembly: SuppressMessage("Usage", "AZC0030:Use one of the following pre-approved namespace groups (https://azure.github.io/azure-sdk/registered_namespaces.html): Azure.AI, Azure.Analytics, Azure.Communication, Azure.Containers, Azure.Core.Expressions, Azure.Data, Azure.DigitalTwins, Azure.Identity, Azure.IoT, Azure.Learn, Azure.Management, Azure.Media, Azure.Messaging, Azure.MixedReality, Azure.Monitor, Azure.ResourceManager, Azure.Search, Azure.Security, Azure.Storage, Azure.Template, Microsoft.Extensions.Azure", Justification = "<Pending>", Scope = "namespaceanddescendants", Target = "~N:Azure.Compute.Batch")]
```

Comment:
`AZC0030` tests for approved suffixes in model names.   Either this isn't necessary, or the associated text is very wrong here.

_[(src)](https://github.com/Azure/azure-sdk-tools/blob/bc423d5abf1169e3b29c019382304f6a3d3a8fe8/src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/ModelName/GeneralSuffixAnalyzer.cs#L23)_

Reply:
Sorry this was suppression i put in before our namespace Azure.Compute.Batch was approved.  I removed this plus all the other suppressions but there was one I had to leave in.  We have this one Class that doesn't comply with the naming conventions, "ExitOptions" that we got approval from the SDK board to leave as it.